javascript/ql/src/Statements/IgnoreArrayResult.qhelp

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+The <code>concat</code>, <code>join</code> and <code>slice</code> methods are
+pure and do not modify any of the inputs or the array the method was called 
```

Comment:
Just a question here. Are you deliberately using the past here? ….was called...

Reply:
no..
I'm changing it to `is called`.